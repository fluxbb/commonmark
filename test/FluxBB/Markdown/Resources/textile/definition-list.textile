- HTML := HyperText Markup Language
- HTML :=
    HyperText
    Markup Language =:
- HTML := HyperText Markup Language