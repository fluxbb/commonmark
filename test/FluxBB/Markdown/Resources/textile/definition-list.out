<dl>
<dt>HTML</dt>
<dd>HyperText Markup Language</dd>
<dt>HTML</dt>
<dd><p>HyperText
Markup Language</p></dd>
<dt>HTML</dt>
<dd>HyperText Markup Language</dd>
</dl>